Source/EventFlow.PostgreSql/EventStores/Scripts/0002 - Create eventdatamodel_list_type.sql

```diff
@@ -0,0 +1,15 @@
+DO $$
```

Comment:
I haven't use PostgreSql, but why is this `+DO $$` here and not in the other files?

Reply:
In PostgreSQL we need to use DO  to execute code blocks in order to execute the 'IF EXIST' check.

https://www.postgresql.org/docs/current/static/sql-do.html